acceptance-tests/src/main/java/io/blueocean/ath/model/MultiBranchPipeline.java

```diff
@@ -12,7 +12,7 @@
 import java.io.IOException;
 import java.util.List;
 
-public class MultiBranchPipeline extends Pipeline{
+public class MultiBranchPipeline extends AbstractPipeline {
```

Comment:
Should this override `isMultibranch()` to return true?

Reply:
yes